cmd/subrouter/sr_reset.go

```
@@ -69,9 +76,17 @@ func (r srRunner) reset(ctx context.Context, args []string) error {
 		return fmt.Errorf("-n must be at least 1")
 	}
 
-	server, ok, err := r.selectedRemoteServer()
-	if err != nil {
-		return err
+	server := srServerConfig{}
```

golangci-lint (2.12.2)

[error] ineffectual assignment to server

(ineffassign)